testsuite/unittests/test_symmetric_connections.sli

```diff
@@ -136,10 +139,10 @@ ResetKernel
 }
 pass_or_die
 
-<< /source [1 5] Range >> GetConnections /fwd Set
-<< /source [6 10] Range >> GetConnections /bck Set
-
 {
+  << /source [1 5] Range >> GetConnections /fwd Set
```

Comment:
This test depends on the previous test to create the network. It would be better to join the previous test with this one. assert is a tighter requirement than pass, so it is not strictly necessary to test first that the Connect call passes and then assert that it generated correct results.